uspv/init.go

```diff
@@ -13,10 +13,8 @@ import (
 	"github.com/mit-dci/lit/lnutil"
 )
 
-// Connect dials out and connects to full nodes.
-func (s *SPVCon) Connect(remoteNode string) error {
+func GetSeedAdrs(s *SPVCon, remoteNode string) error {
```

Comment:
This should be a method on SPVCon; so should the others

Reply:
will do